frontend/src/utils/__tests__/useFetch.test.ts

```diff
@@ -0,0 +1,53 @@
+import { useFetch } from '../useFetch';
+import { renderHook } from '@testing-library/react-hooks';
+
+function fetchMock(url, opts) {
+  return Promise.resolve({
+    json: () =>
+      Promise.resolve({
+        data: opts,
+      }),
```

Comment:
```suggestion
    json: () => {
      Promise.resolve({
        data: opts,
      }) },
```
It might be dangerous to have a `=>` or `return` without braces after it in JS because the language might add a `;` at the end of the line.

Reply:
I guess you meant parentheses. Wrapped.